ws/adapter_test.go

```diff
@@ -0,0 +1,52 @@
+package ws_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/mainflux/mainflux/ws"
+	"github.com/mainflux/mainflux/ws/mocks"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/mainflux/mainflux"
+)
+
+var (
```

Comment:
Please drop the variable grouping. Also, I am not sure about this error, since it is duplicated in both mocks and ws_test. Perhaps we could promote it to the domain.